map/framework.cpp

```diff
@@ -1394,6 +1394,9 @@ string Framework::GetCountryName(m2::PointD const & pt) const
 
 bool Framework::Search(search::SearchParams const & params)
 {
+  if (ParseDrapeDebugCommand(params.m_query))
```

Comment:
Could you please group this check with ParseEditorDebugCommand()? For example, you may create a helper function, say:
```c++
// Returns true when |command| is a debug command.
bool ParseDebugCommand(std::string const & command)
...
```
and call it there.